_posts/2017-03-20-developers-toolbox-q1-2017.md

```diff
@@ -0,0 +1,83 @@
+---
+layout: post
+title: Four Useful Additions to Any Developers Toolbox in Q1 2017
+date: 2017-03-20
+author: Ro Wang
+tags: tips efficiency docker
+featured_image: "/blog/content-authoring-d8.jpg"
+featured_image_alt: "Q1 2017 Tips"
+featured_image_height: "917px"
+featured_image_width: "1474px"
+summary: What we've added to our developer toolboxes in Q1 2017!
+drupal_planet_summary: |
```

Comment:
@lrowang You don't need to include this unless you're including the drupal planet tag - just the regular `summary` is needed